client/components/AdminDashboard.jsx

```diff
@@ -0,0 +1,108 @@
+"use client";
+import React, { useState } from "react";
+import { DataGrid } from "@mui/x-data-grid";
+
+export default function AdminDashboard() {
+  //   const [hasVisited, setHasVisited] = useState(false);
+  const status_options = ["Requested", "Confirmed", "Pending", "Visited"];
+
+  const [rows, setRows] = useState([
+    {
+      id: 1,
+      col1: false,
```

Comment:
I'd suggest these fields should be named according to what they contain. status, timeSlot, firstName etc

Reply:
yea, I was thinking about that but it seemed redundant at the time with the header. But prob better to rename it to a meaningful name like you pointed out.